### DISCUSSION TITLE
Limit of fuel sensor over 100%

### DISCUSSION THREAD

**Discussion by mhrfabio87** [FrSkyRC/ETHOS-Feedback-Community#3295] [Q&A] [answered]
Hi, I need the fuel sensor to exceed the 100% threshold on ethos. This is because I use a flybarless unit (Brain 2) which sends the vibration value on that channel exceeding the 100% limit (it can go up to 1000%). Can it be implemented in the next update?Thank you

**Comment by bsongis-frsky** [comment, marked as answer]
Why Brain sends a vibration value on a ID which is dedicated to fuel level?
It's not like there is no more available IDs!
